restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -361,9 +362,9 @@
 	"projectLogResetLabel": "Reiniciar",

```

Comment:
Also in (342), change for:
"pauseStepLengthTooltip": "Cuántos segundos de espera silenciosa antes de continuar con",